addons/l10n_be/__manifest__.py

```diff
@@ -46,6 +46,7 @@
         'data/account_tax_report_data.xml',
         'data/l10n_be_sequence_data.xml',
         'data/menuitem_data.xml',
+        'data/auto_load_coa.xml',
```

Comment:
@jco-odoo is there any way of adding `xml` data from another module (`account`) because I don't want we ended up loading same file in every module, for other option we can do a `post_init`